djanco_python2::python_snapshots_before_dec2008, 8753, "No such file or directory (os error 2)"
